src/main/java/org/littleshoot/proxy/impl/ProxyUtils.java

```diff
@@ -514,10 +514,16 @@ public static HttpResponse duplicateHttpResponse(HttpResponse originalResponse)
     public static String getHostName() {
         try {
             return InetAddress.getLocalHost().getHostName();
-        } catch (UnknownHostException e) {
-            LOG.warn("Could not lookup localhost", e);
-            return null;
+        } catch (IOException e) {
+            LOG.debug("Ignored exception", e);
```

Comment:
Could we keep the log message and log this at info/warn? That way when the UnknownHostException occurs there is some record in the log of the reason the lookup failed.